src/app/models/chat-group.ts

```diff
@@ -1,8 +1,8 @@
-import { Timestamp } from '@google-cloud/firestore';
+import { FieldValue, Timestamp } from '@google-cloud/firestore';
 
 export interface ChatGroupI {
   id?: string;
   name?: string;
   participantIds: string[];
-  createdAt: Timestamp;
+  createdAt: FieldValue | Timestamp;
```

Comment:
```suggestion
  createdAt: firestore.FieldValue | firestore.Timestamp;
```
This way we don't need to install the extra package. Always seriously consider whether it's necessary to install extra packages.